datafusion/core/src/catalog/information_schema.rs

```diff
@@ -39,26 +41,32 @@ use super::{
     schema::SchemaProvider,
 };
 
+use crate::config::ConfigOptions;
+
 const INFORMATION_SCHEMA: &str = "information_schema";
 const TABLES: &str = "tables";
 const VIEWS: &str = "views";
 const COLUMNS: &str = "columns";
+const SETTINGS: &str = "settings";
```

Comment:
I don't think `settings` is an "offical" information_schema (if there is such a thing) table. Perhaps we could call this table something like `df_settings` to make it clear that this table is a datafusion extension?

https://www.postgresql.org/docs/current/information-schema.html

Reply:
no it's not. postgresl has this
```bash
willy=#  select name, setting from pg_settings limit 1;
          name           | setting 
-------------------------+---------
 allow_system_table_mods | off
(1 row)
```

df_settings sounds good. should we leave it in information_schema, or move it out?

Reply:
> df_settings sounds good. should we leave it in information_schema, or move it out?

I don't have any preference -- whatever is easier at first (`information_schema` seems reasonable to me(

Reply:
@alamb updated

```bash
❯ show tables;
+---------------+--------------------+-------------+------------+
| table_catalog | table_schema       | table_name  | table_type |
+---------------+--------------------+-------------+------------+
| datafusion    | information_schema | tables      | VIEW       |
| datafusion    | information_schema | views       | VIEW       |
| datafusion    | information_schema | columns     | VIEW       |
| datafusion    | information_schema | df_settings | VIEW       |
+---------------+--------------------+-------------+------------+
4 rows in set. Query took 0.003 seconds.


❯ select * from information_schema.df_settings;
+-------------------------------------------------+---------+
| name                                            | setting |
+-------------------------------------------------+---------+
| datafusion.execution.coalesce_batches           | true    |
| datafusion.execution.time_zone                  | UTC     |
| datafusion.explain.physical_plan_only           | false   |
| datafusion.execution.coalesce_target_batch_size | 4096    |
| datafusion.execution.batch_size                 | 8192    |
| datafusion.optimizer.skip_failed_rules          | true    |
| datafusion.optimizer.filter_null_join_keys      | false   |
| datafusion.explain.logical_plan_only            | false   |
+-------------------------------------------------+---------+
8 rows in set. Query took 0.002 seconds.
```